extensions/reactive-pg-client/runtime/src/test/java/io/quarkus/reactive/pg/client/runtime/PgPoolProducerTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.quarkus.reactive.pg.client.runtime;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.Assert.assertThat;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.vertx.core.Vertx;
+
+public class PgPoolProducerTest {
+
+    private Vertx vertx;
+    private PgPoolTemplate template;
+    private PgPoolProducer producer;
+
+    @Before
```

Comment:
We should really be using JUnit5. Also I think this should be using @QuarkusUnitTest to actually test this in a real quarkus deployment.

Reply:
Is it possible to customize the config in a `@QuarkusUnitTest`?